packages/components/src/listBox/docs/ListBox.stories.tsx

```diff
@@ -0,0 +1,459 @@
+ * Typically, this context is determined by the user’s current selection prior to invoking the menu. 
+ * Listbox can be opened from components such as Selects or Buttons.
+ *
+ * [View repository](https://github.com/gsoft-inc/wl-hopper/tree/main/packages/components/src/ListBox/src)
+ * -
+ * [View package](https://www.npmjs.com/package/@hopper-ui/components)
+ * -
+ * View storybook TODO
+ */
+const meta = {
+    title: "Docs/ListBox",
+    tags: ["autodocs"],
+    parameters: {
+        // Disables Chromatic's snapshotting on documentation stories
+        chromatic: { disableSnapshot: true }
+    },
+    component: ListBox
+} satisfies Meta<typeof ListBox>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+interface ListItemProps {
+    id: number | string;
+    name: string;
+}
+
+interface ListSectionProps {
+    name: string;
+    children: ListItemProps[];
+}
+
+/**
+ * The default ListBox.
+ */
+export const Default = {
+    render: args => {
+        return (
+            <ListBox {...args} aria-label="list of options">
+                <ListBoxItem>Item 1</ListBoxItem>
+                <ListBoxItem>Item 2</ListBoxItem>
+                <ListBoxItem>Item 3</ListBoxItem>
+            </ListBox>
+        );
+    }
+} satisfies Story;
+
+/**
+ * ListBox items with a description.
+ */
+export const Description = {
+    render: args => {
+        return (
+            <ListBox {...args} aria-label="list of options with a description">
+                <ListBoxItem textValue="Item 1">
+                    <Text>Item 1</Text>
+                    <Text slot="description">Description of item 1</Text>
+                </ListBoxItem>
+                <ListBoxItem textValue="Item 2">
+                    <Text>Item 2</Text>
+                    <Text slot="description">Description of item 2</Text>
+                </ListBoxItem>
+                <ListBoxItem textValue="Item 3">
+                    <Text>Item 3</Text>
+                    <Text slot="description">Description of item 3 will be a long one to show wrapping.</Text>
+                </ListBoxItem>
+            </ListBox>
+        );
+    }
+} satisfies Story;
+
+/**
+ * ListBox that is fluid, meaning the width is 100% of its container.
+ */
+export const Fluid = {
+    ...Description,
+    args: {
+        isFluid: true
+    }
+} satisfies Story;
+
+/**
+ * A ListBox can have a single selected item.
+ */
+export const SingleSelection = {
+    render: function Render(args) {
+        const [selectedKeys, setSelectedKeys] = useState<Selection>(new Set(["1"]));
+
+        return (
+            <ListBox {...args}
+                aria-label="list of options"
+                selectedKeys={selectedKeys}
+                onSelectionChange={setSelectedKeys}
+            >
+                <ListBoxItem textValue="Item 1" id="1">
```

Comment:
is TextValue required on every items?

Reply:
yes. well every item that isn't text only, which is pretty much all the time since we put things in a Text component.